src/reference/burp_vs_caido.md

```diff
@@ -0,0 +1,79 @@
+# Burp Suite vs Caido
+
```

Comment:
We should __always__ add an initial description under the main title explaining what this page is for.

This gives context to the user on what this page is about.

For example:

```
This page offers a side-by-side reference of common Burp Suite features and where to find their counterparts in Caido.
```